src/components/Heading/__snapshots__/Heading.spec.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Heading should render as h1 element, when passed "h1" for the element prop 1`] = `
+exports[`Heading should render as h1 element, when passed "%s" for the element prop 1`] = `
```

Comment:
Ah, I forgot that [the placeholders are positional](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout), meaning the param can only be used once. I'd simply shorten the description to: `"should render as %s element"`

Reply:
Nice catch! 🦅 Done in 3624ae9